http_parser.c

```diff
@@ -472,6 +483,89 @@ static struct {
 
 int http_message_needs_eof(const http_parser *parser);
 
+const char* find_crlf(const char* p, const char* data, size_t len);
+
+#if defined(_MSC_VER) // SSE2 is baseline
+#include <intrin.h>
+#define __builtin_ctz _tzcnt_u32
+#elif defined(__SSE2__) && defined(__GNUC__)
+#include <emmintrin.h>
+#endif
+
+#ifndef USE_INTRISICS_CRLF
+# define USE_INTRISICS_CRLF defined(_MSC_VER) || (defined(__SSE2__) && defined(__GNUC__))
```

Comment:
Long line, please wrap it at 80 columns.  (Also applies to any other long lines this PR may introduce, I won't point them out individually.)

As well, this would be better written as:
```c
#if defined(_MSC_VER) || (defined(__SSE2__) && defined(__GNUC__))
# define USE_INTRISICS_CRLF 1
#endif
```

Reply:
It was modified to this to allow overridable defines, as requested by @mscdex earlier

Reply:
Okay, but defining it like this doesn't work, the `#if USE_INTRISICS_CRLF` is always going to evaluate to true.  Try changing it to `#define USE_INTRISICS_CRLF defined(florp)` and you'll see it still evaluates to true.